fixedwidth/src/main/java/org/apache/metamodel/fixedwidth/FixedWidthConfiguration.java

```diff
@@ -33,66 +33,74 @@
 /**
  * Configuration of metadata about a fixed width values datacontext.
  */
-public final class FixedWidthConfiguration extends BaseObject implements
-		Serializable {
+public final class FixedWidthConfiguration extends BaseObject implements Serializable {
 
-	private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 1L;
 
-	public static final int NO_COLUMN_NAME_LINE = 0;
-	public static final int DEFAULT_COLUMN_NAME_LINE = 1;
+    public static final int NO_COLUMN_NAME_LINE = 0;
+    public static final int DEFAULT_COLUMN_NAME_LINE = 1;
 
-	private final String encoding;
-	private final int fixedValueWidth;
-	private final int[] valueWidths;
-	private final int columnNameLineNumber;
-	private final boolean failOnInconsistentLineWidth;
-	private final ColumnNamingStrategy columnNamingStrategy;
+    private final String encoding;
+    private final int fixedValueWidth;
+    private final int[] valueWidths;
+    private final int columnNameLineNumber;
+    private final boolean failOnInconsistentLineWidth;
+    private final boolean headerPresent;

```

Comment:
This new property is redundant (and thus potentially 'dirty') because `columnNameLineNumber` holds the same metadata. If it is 0 (`NO_COLUMN_NAME_LINE `), then it means that there is no header.